tests/accesscontrol/suite.go

```diff
@@ -275,7 +295,7 @@ func checkForbiddenCapability(containers []*provider.Container, capability strin
 		switch {
 		case cut.SecurityContext == nil:
 		case cut.SecurityContext.Capabilities == nil:
-		case strings.Contains(cut.SecurityContext.Capabilities.String(), capability):
+		case isContainerCapabilitySet(cut.SecurityContext.Capabilities, capability):
```

Comment:
The security context in the pod should be checked first and overridden by the security context in the container
see:
- https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-container
- https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-pod

Reply:
@edcdavid This is true for other fields of the securityContext, but not for this one. The capabilities can only be set at [container level](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.30/#securitycontext-v1-core), not at [pod level](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.30/#podsecuritycontext-v1-core).